gapis/trace/android/trace.go

```diff
@@ -37,11 +44,25 @@ import (
 	"github.com/google/gapid/gapidapk"
 	"github.com/google/gapid/gapidapk/pkginfo"
 	gapii "github.com/google/gapid/gapii/client"
-	perfetto "github.com/google/gapid/gapis/perfetto/android"
+	"github.com/google/gapid/gapis/perfetto"
+	perfetto_android "github.com/google/gapid/gapis/perfetto/android"
 	"github.com/google/gapid/gapis/service"
+	"github.com/google/gapid/gapis/trace/android/adreno"
+	"github.com/google/gapid/gapis/trace/android/validate"
 	"github.com/google/gapid/gapis/trace/tracer"
 )
 
+const (
+	activityName                            = "com.samples.cube.VKCubeActivity"
+	intentAction                            = "android.intent.action.MAIN"
+	validationPackageName                   = "com.samples.cube"
+	bufferSizeKb                            = uint32(131072)
+	counterPeriodNs                         = uint64(50000000)
+	durationMs                              = 7000
+	gpuCountersDataSourceDescriptorName     = "gpu.counters"
+	gpuRenderStagesDataSourceDescriptorName = "gpu.renderstages"
```

Comment:
go is all about short names. I don't really care, but these are getting a bit long.... :)

Reply:
Please forgive me to ack this one. I am still struggling with abbreviation :/